GVRf/Framework/framework/src/main/java/org/gearvrf/GVRScene.java

```diff
@@ -113,6 +112,25 @@ private GVRScene(GVRContext gvrContext, long ptr) {
     }
 
     /**
+     * Sets the useMultiview flag in scene
```

Comment:
We should find a better place for the flag; a place that is completely hidden from end users.